typescript/src/_copy_me_fvarela/StringCalculator.test.ts

```diff
@@ -1,14 +1,14 @@
 import { StringCalculator } from "./StringCalculator";
 
 describe(StringCalculator, () => {
-  it("should return 0 when the input is an empty string", () => {
+  it("`An empty string returns zero `'' => 0`", () => {
```

Comment:
Un comentario menor, en Jest `it` vendría a representar lo que señala `describe`, por lo que el nombre completo de este test se leería cómo:

> StringCalculator An empty string returns zero